lib/ex_doc/formatter/html/templates/module_template.eex

```diff
@@ -13,7 +13,27 @@
     <script type="text/javascript" charset="utf-8" src="js/jquery.js"></script>
     <script type="text/javascript" charset="utf-8" src="js/app.js"></script>
     <script type="text/javascript" charset="utf-8" src="js/highlight.pack.js"></script>
-    <script>hljs.initHighlightingOnLoad();</script>
+    <script type="text/javascript" charset="utf-8">
+      hljs.initHighlightingOnLoad();
+      // languages supported: 'bash', 'coffeescript', 'css', 'diff', 'elixir', 'erlang', 'erlang-repl', 'http', 'javascript', 'json', 'markdown', 'sql', 'xml'
```

Comment:
It's because i don't know where to put the information about the aliases.
We should add it to the documentation, or README.md, ... I 'm clueless where to add it to

      /* languages supported and their aliases
        bash, sh, zsh,
        coffeescript, coffee, cson, iced,
        css,
        diff, patch,
        elixir, iex,
        erlang, erl,
        erlang-repl,
        http,
        javascript, js,
        json,
        markdown, md, mkdown, mkd,
        sql,
        xml, html, xhtml, rss, atom, xsl, plist,
      */